config/data.js

```diff
@@ -1738,6 +1739,7 @@ module.exports = {
       y_2016: 4.8,
       y_2017: 5.0,
       y_2018: 5.2,
+      y_2019: 
```

Comment:
@Vanie2020 there seems to be a value missing here?

Reply:
I have adjusted it and am pushing again now.  Thank you for feedback

Reply:
Ok I see what happened.  That was Work from Home on line 1742 which don't apply to this branch.  The changes I was originally pushing on this branch apply to line 73 which is Bike to Work.  Can u disregard line 1742 or do i need to repush?  Sorta still confused at how I can change a particular value and the config file will throw in an empty space of another value which im not working on into the branch.  Anyways let me know how to move forward on this one.

Reply:
Ahh I see. I can remove this line from the commit in Github for now I think so we can keep this moving. Will hopefully be easier to avoid stuff like this in the future once you've got a little bit more git experience.